go2_robot_sdk/launch/webrtc_web.launch.py

```
@@ -0,0 +1,115 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+
+
+def generate_launch_description():
+    # Declare launch parameters
+    robot_ip = LaunchConfiguration('robot_ip', default=os.getenv(
+        'ROBOT_IP', os.getenv('GO2_IP', '')))
+    enable_video = LaunchConfiguration('enable_video', default='true')
+    urdf_file_name = LaunchConfiguration('urdf_file_name', default='go2.urdf')
```

Ruff (0.8.2)

Local variable `urdf_file_name` is assigned to but never used

Remove assignment to unused variable `urdf_file_name`

(F841)